src/Types/Input/InputObjectType.hack

```diff
@@ -5,7 +5,8 @@ use namespace Slack\GraphQL;
 use namespace Graphpinator\Parser\Value;
 
 <<__ConsistentConstruct>>
-abstract class InputObjectType extends NamedInputType {
+abstract class InputObjectType extends NamedType {
+    use TNamedInputType;
```

Comment:
When we have a combination of interface + matching trait, we need something to distinguish the two, so I went with the I prefix for the interface and T for the trait. I can rename it to whatever you like (e.g. one of them could have no prefix).

Reply:
I don't have a preference. Internally we suffix types with `_t`. When I was doing https://github.com/slackhq/hack-json-schema I had read through a couple of the open source libs from facebooked and learned ya'll prefix types with `T` and interfaces with `I` which I've liked.

Reply:
It might be a bit confusing that we use the T prefix for both traits and for generic type params/type constants.

IIRC the T prefix is actually required by Hack for type constants and/or generic type params, but maybe that's changed or I remember wrong.